CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## v0.2.0 (2017-06-16)
+
+* Default listen port changed to 9216.
+* All log messages now go to stderr. Logging flags changed.
+* Fewer messages on default INFO logging level.
```

Comment:
I'm not sure about this one. If you disable logs for requests (because for every scrap request there was a log message) then you won't have information that exporter is processing them? Unless exporter is also instrumented and it delivers info about processed requests? I would like to avoid situation where exporter brakes but we won't know because we don't have any insights.